utils/scripts/terraform/openstack/README.md

```diff
@@ -0,0 +1,21 @@
+# Provision vm in openstack
+Script to provision vm in openstack using terraform
```

Comment:
You can add an usage example, like adding a node for X

Reply:
Done.